### ISSUE TITLE
Please document user privacy extensively

### ISSUE THREAD

**Issue by charles-plessy** [Mailtrain-org/mailtrain#1361]
Hello,

I am a French citizen residing in a foreign country, and for that reason my consulate shares my email address to political parties so that they can inform me.  Most of them use mailing companies for their campaigns, but at least one of them appears to use a self-hosted mailtrain server.

I  think that the landing page of mailtrain servers should display a prominent "Privacy" that informs the targets of the communication about for instance the following:

 - in which jurisdiction is this server running? (geolocalisation of IP address, ...)?
 - is click tracking enabled for some campaigns on this server?
 - what exactly is recorded by the click tracking feature and all other privacy-sensitive features?
 - what is the password / authentication policy for the administrators entrusted with personal email data?
 - can the click history of single individuals be browsed by administrators?
 - how long is this personal data stored?
 - can the click history of single individuals be extracted from database, logs, etc,. by skilled administrators or attackers?
 - how difficult is it for an attacker to steal personal information mailtrain server? (please provide scenarios for each level of privilege of access).
 - if a target of communication unsubscribes, is it guaranteed that every personal information (in the sense of GPDR) about this person deleted if the mailtrain server runs pristine code? How about backups?
 - if a target of communication is re-added (in my case, at the next election), will the data be aggregated?
 - are there mechanisms in place that would make it impossible on purpose the administrators to provide personal data other than the email addresses to law enforcement authorities ? (Just imagine the winners of one side wanting to identify the supporter of the other side…).

In conclusion, although my issue may sound a bit demanding, I am really happy that mailtrain exists, because I hope it will empower the targets to engage in a constructive negotiation with the senders and the state services that provide them with personal data about what privacy level should be set (in particular regarding disabling click tracking).

Thanks for your patience, and all the best for your project!

Charles

**Comment by forgeOneHQ** [comment]
We are going to start with the development and testing of mailtrain v3 in the next weeks.

You are welcome to help us with the testing as soon as the first release candidate is available.
